site/app/templates/course/EditCourseMaterialsForm.twig

```diff
@@ -0,0 +1,116 @@
+        </label>
+        <input type="hidden" name="csrf_token" value="{{ csrf_token }}" />
+    </div>
+    <script type="text/javascript">
+        function submitEdit(){ // Submit button
+                var sectionsEdit = [];
+                var cmTime = document.getElementById("edit_picker").value;
+                var requestedPath = document.getElementById("material-edit-form").getAttribute("data-directory");
+                var hideFromStudents = 'off';
+                if  ($("#hide-materials-checkbox-edit").is(':checked'))  {
+                    hideFromStudents = 'on';
+                }
+                
+                var reg_sectionArray = JSON.parse('{{ reg_sections|json_encode|raw }}');
+                reg_sectionArray.forEach(function(element){
+                    var currentSectionEdit = '#section-edit-' +element['sections_registration_id']
+                    if  ($(currentSectionEdit).is(':checked'))  {
+                        sectionsEdit.push(element['sections_registration_id']);
+                    }
+                });
+                if({{ reg_sections|json_encode|raw }} == null || $("#all_Sections_Showing_No").is(':checked') == true){
+                    sectionsEdit = null;
+                }
+                handleEditCourseMaterials("{{ csrf_token }}", hideFromStudents, requestedPath, sectionsEdit, cmTime);
+        };
+        
+        flatpickr('.edit_picker', {
+                    plugins: [ShortcutButtonsPlugin(
+                        {
+                            button: [
+                                {
+                                    label: "Now"
+                                },
+                                {
+                                    label: "Tomorrow"
+                                },
+                                {
+                                    label: "End of time"
+                                }
+
+                            ],
+                            onClick: (index, fp) => {
+                                switch (index) {
+                                    case 0:
+                                        updateToServerTime(fp);
+                                        break;
+                                    case 1:
+                                        updateToTomorrowServerTime(fp);
+                                        break;
+                                    case 2:
+                                        date = new Date("9998-01-01 00:00:00");
+                                        fp.setDate(date,true);
+                                        break;
+                                }
+                            }
+                        }
+                    )],
+                    allowInput: true,
+                    enableTime: true,
+                    enableSeconds: true,
+                    time_24hr: true,
+                    dateFormat: "Y-m-d H:i:S"
+                    });
```

Comment:
Can you fix the spacing here where the first indentation is 12 characters instead of 4. (same with the control structure above this one). Also make this closing `});` equivalent indentation with the starting line:
```
    flatpicker...({
    });
```